clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/main/MainIngestionMonitorService.java

```diff
@@ -118,7 +114,7 @@ public boolean startInner() throws Exception {
     serverFuture = bootstrap.bind(applicationPort).sync();
     LOGGER.info("Report listener service started on port: {}", applicationPort);
     heartbeatTimeoutMs = configLoader.getCombinedProperties()
-        .getLong(SERVER_INGESTION_ISOLATION_HEARTBEAT_TIMEOUT_MS, 180 * Time.MS_PER_SECOND);
+        .getLong(SERVER_INGESTION_ISOLATION_CONNECTION_TIMEOUT_MS, 180 * Time.MS_PER_SECOND);
```

Comment:
Is this intended? connection timeout into HB timeout?

Reply:
Yes, I changed the name of the config. So this means if two processes does no have heartbeat refresh in this time period, it is considered connection lost and we should perform kill and restarting of the fork process.